c7n/filters/core.py

```diff
@@ -1095,9 +1097,38 @@ def process(self, resources, event=None, filter=Filter):
                                     f"Error trying to serialize {CELEvalError(*e.args)}"
                                 )
                                 filtered_resources.append(None)
-
             return filtered_resources
 
+        def get_accounts_cross_account_access(self, resource, accounts):
```

Comment:
Possibly rename this to `get_cross_account_access()` and pass in checker_field as an argument?

vs. keeping as-is to make things more explicit for policy authors?